easybuild/tools/toolchain/compiler.py

```diff
@@ -41,6 +42,9 @@
 # by doing eb --optarch=GENERIC
 OPTARCH_GENERIC = 'GENERIC'
 
+# Characters that separate toolchains and options in --optarch
+OPTARCH_TOOLCHAIN_DELIMITER = ';'
+OPTARCH_OPTION_DELIMITER = ':'
```

Comment:
I would shorten these a bit, e.g. `OPTARCH_SEP` for `;` and `OPTARCH_MAP_CHAR` for `:`